Escargo/Assets/Scripts/SnaillingScript.cs

```diff
@@ -80,20 +80,41 @@ void Start()
             snaillings[currentSnail].transform.position = new Vector3(snaillings[currentSnail].transform.position.x,
                 snaillings[currentSnail].transform.position.y, 0);
             snaillings[currentSnail].tag = "P" + playerID.ToString() + "Snailling";
-            //findPath(currentSnail, (int)GetComponent<Transform>().position.x, (int)GetComponent<Transform>().position.y);
             currentSnail++;
         }
         /* snailings move */
         moveTimer += Time.deltaTime;
-        if (moveTimer >= 1.0f)
+        if (moveTimer >= 1.5f)
         {
             moveTimer = 0f;
             for (int i = 0; i < currentSnail; i++)
             {
-                if (snaillings[i] != null && snaillingsMove[i].Count > 0)
+                int oX = (int)snaillings[i].transform.position.x;
```

Comment:
This line will cause an runtime error if the snailling is null. Snailling becomes null when they reach the finish line.